Helper/Variation.php

```
@@ -111,7 +115,7 @@ public function isDefaultVariationCode($code)
             if ($code === $this->getDefaultVariationCode()) {
                 return true;
             }
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
```

Why is the exception swallowed?